UserComponents/BlazorRepl.UserComponents.csproj

```diff
@@ -7,6 +7,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.1.5" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
```

Comment:
Since this is just a container assembly that is being replaced on the fly during app usage, we don't need to have analyzers installed in it for now.